source/slang/glsl.meta.slang

```diff
@@ -4615,4 +4574,607 @@ __spirv_version(1.3) [require(spirv)]
 {
     shader_subgroup_preamble<T>();
     return QuadReadAcrossDiagonal(value);
+}
+
+//// GLSL atomic
+
+// this is actaully just a fancy storage buffer, and will be 
+// converted into one as such
```

Comment:
You should try to stay in "impartiality tone", which is objective and formal.
I would say more like as following,
```
// atomic_uint is internally implemented with Shader Storage Object Buffer.
```